test/fixtures/rules/std_rule_cases/L026.yml

```diff
@@ -193,3 +193,10 @@ test_exasol_invalid_foreign_key_from:
   configs:
     core:
       dialect: exasol
+
+test_tsql_select_literal_instead_of_identifier:
```

Comment:
Should we consider adding similar test cases to other rules such as L012, L013, L014, and/or L029? IIUC, L026 does not specifically relate to column aliases, while those other rules do.

Reply:
Well to be honest that test case is no longer valid. Will leave it in there (can never have to many test cases!) but will rename.

I'll add to L044 as that's the other one that was flagging with this. The rest didn't raise.